manifests/template.pp

```diff
@@ -27,11 +27,13 @@
 # Copyright 2015  Vladislav Tkatchev
 #
 define zabbix::template (
-  $templ_name   = $title,
-  $templ_source = '',
+  $templ_name     = $title,
+  $templ_source   = '',
+  $zabbix_version = $zabbix::params::zabbix_version,
```

Comment:
thanks for the PR! Can you pleae add a datatype here? I guess a simple string is okay:
```suggestion
  String[1] $zabbix_version = $zabbix::params::zabbix_version,
```

Reply:
please also add some documentation for the new parameter in the header.

Reply:
Sure!

Reply:
Done. You can cancel some runs if you want to btw :stuck_out_tongue: